packages/amplify-graphql-model-transformer/src/resources/rds-model-resource-generator.ts

```diff
@@ -16,8 +16,9 @@ import { Fn } from 'aws-cdk-lib';
 import { SubscriptionFilter } from 'aws-cdk-lib/aws-sns';
 
 export const RDS_STACK_NAME = 'RdsApiStack';
-// TODO: Need to change this to production SNS topic
-const RDS_PATCHING_SNS_TOPIC_ARN = 'arn:aws:sns:us-east-1:956468067974:AmplifyRDSLayerNotification';
+// Beta SNS topic - 'arn:aws:sns:us-east-1:956468067974:AmplifyRDSLayerNotification'
+// PROD SNS topic - 'arn:aws:sns:us-east-1:582037449441:AmplifyRDSLayerNotification'
```

Comment:
Probably don't want to vend this info here.

Reply:
Thought it would be handy to have this information here. Just in case someone checkin beta ARN, reviewer can use this as quick reference.